### PR TITLE
Alternative reskin: Gives maintainers/coders lusty xenomorph alien queen reskin if they happen to be the queen (I think all of the rest of these will be public)

### PR DESCRIPTION
**Pull request by Militaires**
I'm making this as a pr to just throw it in as I already made the sprites after years of tears and blood and pain. I think I'll just have all of the rest of them be public since this is a pain in the ass.

_I added myself too cause I gotta get something outta this ;)_
EDIT: FUCK

### CODE REVIEW

**Comment by WJohn** [comment]
how am i not on that list

**Comment by Razharas** [comment]
>Actual ckeys in the code

At least make it load from the config file for fucks sake

**Comment by as334** [comment]
I thought we're not allowed to have this sort of special club thing in the code?(I will retract this statement if you add the ckey as334 to the list)

**Comment by Militaires** [comment]
@as334 done in next commit

:)

**Comment by Niknakflak** [comment]
If this is merged, I wi personally gib anyone using this

File: code/modules/mob/living/carbon/alien/humanoid/caste/praetorian.dm [hunk 1]
```diff
@@ -38,10 +38,17 @@
...
-	if(!alien_type_present(/mob/living/carbon/alien/humanoid/royal/queen))
+	if(!alien_type_present(/mob/living/carbon/alien/humanoid/royal/queen) && (!(user.ckey in list("optimumtact", "coiax", "technoalchemist", "crimsonvision", "misterperson", "mrstonedone", "jordie0608", "xhuis", "iamgoofball", "cheridan", "pkpenguin321", "basilman"))))
 		var/mob/living/carbon/alien/humanoid/royal/queen/new_xeno = new (user.loc)
 		user.alien_evolve(new_xeno)
 		return 1
 	else
-		user << "<span class='notice'>We already have an alive queen.</span>"
-		return 0
\ No newline at end of file
+		if(user.ckey in list("optimumtact", "coiax", "technoalchemist", "crimsonvision", "misterperson", "mrstonedone", "jordie0608", "xhuis", "iamgoofball", "cheridan", "pkpenguin321", "basilman"))
```
Comment (xxalpha) [inline code/modules/mob/living/carbon/alien/humanoid/caste/praetorian.dm]: I am deeply offended by this list.

Also this isn't allowed.

Comment (KorPhaeron) [inline code/modules/mob/living/carbon/alien/humanoid/caste/praetorian.dm]: yikes

**Comment by GunHog** [comment]
I know this is a meme PR, but no WJ?

File: code/modules/mob/living/carbon/alien/humanoid/caste/praetorian.dm [hunk 2]
```diff
@@ -38,10 +38,17 @@
 	if(node.recent_queen_death)
 		user << "<span class='danger'>You are still too burdened with guilt to evolve into a queen.</span>"
 		return 0
-	if(!alien_type_present(/mob/living/carbon/alien/humanoid/royal/queen))
+	if(!alien_type_present(/mob/living/carbon/alien/humanoid/royal/queen) && (!(user.ckey in list("optimumtact", "coiax", "technoalchemist", "crimsonvision", "misterperson", "mrstonedone", "jordie0608", "xhuis", "iamgoofball", "cheridan", "pkpenguin321", "basilman"))))
 		var/mob/living/carbon/alien/humanoid/royal/queen/new_xeno = new (user.loc)
 		user.alien_evolve(new_xeno)
 		return 1
 	else
```
Comment (lordpidey) [inline code/modules/mob/living/carbon/alien/humanoid/caste/praetorian.dm]: The way this if statement is setup, one of the people on this list could become a queen even if there is already one.

File: code/modules/mob/living/carbon/alien/humanoid/caste/praetorian.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/modules/mob/living/carbon/alien/humanoid/caste/praetorian.dm]: what the fuck why am i in there

**Comment by PKPenguin321** [comment]
this is some next level shit

**Comment by Niknakflak** [comment]
Please dont reopen a new pr

**Pull request closed without merging**